### PR TITLE
Follow Topic

### PR DESCRIPTION
**Pull request by tienquanptit**

### CODE REVIEW

File: app/Http/Controllers/User/FollowController.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    }
+
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param  int $id
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/FollowController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/FollowController.php</br>**Line**: 97</br>Expected 1 space between "'follow_id'" and double arrow; 0 found

File: app/Models/Topic.php [hunk 2]
```diff
@@ -63,4 +64,15 @@ public function follows()
     {
         return $this->morphMany('App\Models\Follow', 'follow');
     }
+
+//    follow topic theo user
+    public function followed($topicId){
```
Comment (framgiaciv3) [inline app/Models/Topic.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/Topic.php</br>**Line**: 69</br>Opening brace should be on a new line

File: app/Http/Controllers/User/FollowController.php [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        $follow = Follow::where([
+            'follow_id' => $id,
+            'user_id' => Auth::id(),
+            'follow_type' => 'App\Models\Topic',
+        ])->first();
+        $follow->delete();
```
Comment (BunnyPi04) [inline app/Http/Controllers/User/FollowController.php]: sử dụng repository

File: app/Repositories/Contracts/FollowRepository.php [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?php
```
Comment (framgiaciv3) [inline app/Repositories/Contracts/FollowRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.NamespaceDeclaration.BlankLineAfter</br>**Severity**: error</br>**File**: app/Repositories/Contracts/FollowRepository.php</br>**Line**: 3</br>There must be one blank line after the namespace declaration

File: app/Repositories/Eloquents/FollowEloquentRepository.php [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (framgiaciv3) [inline app/Repositories/Eloquents/FollowEloquentRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.NamespaceDeclaration.BlankLineAfter</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/FollowEloquentRepository.php</br>**Line**: 3</br>There must be one blank line after the namespace declaration

**Pull request merged**